src/test/java/loanbook/logic/commands/SetEmailCommandTest.java

```diff
@@ -30,112 +31,88 @@
     @Test
     public void constructor_nullOldEmail_throwsNullPointerException() {
         thrown.expect(NullPointerException.class);
-        new SetEmailCommand(null, VALID_USER_EMAIL1);
+        new SetEmailCommand(null, new Password("a12345"));
     }
 
     @Test
     public void constructor_nullNewEmail_throwsNullPointerException() {
         thrown.expect(NullPointerException.class);
-        new SetEmailCommand(VALID_USER_EMAIL2, null);
+        new SetEmailCommand(new Email(VALID_USER_EMAIL2), null);
```

Comment:
Here, you have a null `Password` instead of a null `newEmail`.

Perhaps change it to `new SetEmailCommand(null, new Password("a12345"));`